oadp-operator/controllers/restic.go

```diff
@@ -1,14 +1,263 @@
+					RunAsUser:          pointer.Int64(0),
+					SupplementalGroups: []int64{},
+				},
+				Volumes: []v1.Volume{
+					// Cloud Provider volumes are dynamically added in the for loop below
+					{
+						Name: "host-pods",
+						VolumeSource: v1.VolumeSource{
+							HostPath: &v1.HostPathVolumeSource{
+								Path: resticPvHostPath,
+							},
+						},
+					},
+					{
+						Name: "scratch",
+						VolumeSource: v1.VolumeSource{
+							EmptyDir: &v1.EmptyDirVolumeSource{},
+						},
+					},
+					{
+						Name: "certs",
+						VolumeSource: v1.VolumeSource{
+							EmptyDir: &v1.EmptyDirVolumeSource{},
+						},
+					},
+				},
+				Tolerations: velero.Spec.ResticTolerations,
+				Containers: []v1.Container{
+					{
+						Name: "velero",
+						SecurityContext: &v1.SecurityContext{
+							Privileged: pointer.Bool(true),
+						},
+						Image: fmt.Sprintf("%v/%v/%v:%v", os.Getenv("REGISTRY"), os.Getenv("PROJECT"), os.Getenv("VELERO_REPO"), os.Getenv("VELERO_TAG")),
+						// velero_image_fqin: "{{ velero_image }}:{{ velero_version }}"
+						// velero_image: "{{ registry }}/{{ project }}/{{ velero_repo }}"
+						// velero_version: "{{ lookup( 'env', 'VELERO_TAG') }}"
+						ImagePullPolicy: "Always",
+						Resources:       r.getVeleroResourceReqs(velero), //setting default.
+						Command: []string{
+							"/velero",
+						},
+						Args: []string{
+							"restic",
+							"server",
+						},
+						VolumeMounts: []v1.VolumeMount{
+							{
+								Name:             "host-pods",
+								MountPath:        "/host_pods",
+								MountPropagation: &mountPropagationToHostContainer,
+							},
+							{
+								Name:      "scratch",
+								MountPath: "/scratch",
+							},
+							{
+								Name:      "certs",
+								MountPath: "/etc/ssl/certs",
+							},
+						},
+						Env: []v1.EnvVar{
+							{
+								Name:  "HTTP_PROXY",
+								Value: os.Getenv("HTTP_PROXY"),
+							},
+							{
+								Name:  "HTTPS_PROXY",
+								Value: os.Getenv("HTTPS_PROXY"),
+							},
+							{
+								Name:  "NO_PROXY",
+								Value: os.Getenv("NO_PROXY"),
+							},
+							{
+								Name: "NODE_NAME",
+								ValueFrom: &v1.EnvVarSource{
+									FieldRef: &v1.ObjectFieldSelector{
+										FieldPath: "spec.nodeName",
+									},
+								},
+							},
+							{
+								Name: "POD_NAME",
+								ValueFrom: &v1.EnvVarSource{
+									FieldRef: &v1.ObjectFieldSelector{
+										FieldPath: "metadata.name"},
+								},
+							},
+							{
+								Name: "VELERO_NAMESPACE",
+								ValueFrom: &v1.EnvVarSource{
+									FieldRef: &v1.ObjectFieldSelector{
+										FieldPath: "metadata.namespace",
+									},
+								},
+							},
+							{
+								Name: "VELERO_SCRATCH_DIR",
+								ValueFrom: &v1.EnvVarSource{
+									FieldRef: &v1.ObjectFieldSelector{
+										FieldPath: "/scratch",
+									},
+								},
+							},
+						},
+					},
+				},
+				InitContainers: []v1.Container{
```

Comment:
When we looked at this last week the conclusion we came to was this is no longerr needed, so we can remove this initContainer.